build.sbt

```diff
@@ -11,12 +11,13 @@ lazy val root = (project in file(".")).
     )),
     name := "scalatest-example",
     Compile / PB.targets := Seq(
-      scalapb.gen() -> (Compile / sourceManaged).value / "scalapb"
+      scalapb.gen(FlatPackage) -> (Compile / sourceManaged).value / "scalapb",
```

Comment:
The generator parameters of `scalapb.gen` and `scalapb.validate.gen` must match, otherwise they won't be able to reference each other correctly.